tests/test_parsers.py

```diff
@@ -245,3 +246,20 @@ def test_parse_obographs_merged(self):
             )
         msdf = parse_sssom_table(outfile)
         self.assertTrue(custom_curie_map.items() <= msdf.prefix_map.items())
+
+    def test_tsv_to_json_and_back(self):
+        """Test converting SSSOM TSV => JSON => SSSOM TSV such that it is reproducible."""
+        sample_tsv = f"{test_data_dir}/sample1.sssom.tsv"
+        json_outfile = f"{test_out_dir}/sample1.json"
+        msdf1 = parse_sssom_table(sample_tsv)
```

Comment:
@hrshdhgd other hints: when writing tests, make the things you're testing more explicit. The way this test is written, it doesn't really test anythign specific. Why not explicitly check that `ORCID` is in the places you expect to to be? Otherwise, someone reading this test learns nothing about what's actually supposed to be happening here.